packages/kernel-5.10/Cargo.toml

```diff
@@ -15,11 +15,13 @@ path = "../packages.rs"
 # Use latest-kernel-srpm-url.sh to get this.
 url = "https://cdn.amazonlinux.com/blobstore/0af5f80d00a3d5a867d4959d74751bc7d24b1bcb0ab8a5de558ae301ae0fa52e/kernel-5.10.228-219.884.amzn2.src.rpm"
```

Comment:
If we are going to do this, there are two other packages for which we use Amazon Linux's upstream:

- grub
- libkcapi